EventListener/InitDateListener.php

```diff
@@ -9,19 +9,25 @@
 class InitDateListener
 {
     private $dm;
-    private $inteval;
+    private $interval;
+    private $days;
 
-    public function __construct(DocumentManager $documentManager, $inteval = 365)
+    public function __construct(DocumentManager $documentManager, $interval = 365, $days)
```

Comment:
`$days`? What is `$days`?

https://github.com/teltek/PuMuKIT2-expired-video-bundle/blob/master/Resources/config/services.xml#L21